crates/wash-lib/src/cli/capture.rs

```diff
@@ -76,36 +68,42 @@ pub async fn handle_replay_command(cmd: CaptureReplayCommand) -> Result<CommandO
     let capture = ReadCapture::load(cmd.capture_file_path).await?;
 
     let filtered = capture.messages.into_iter().filter_map(|msg| {
-        let mut inv: Invocation = rmp_serde::from_slice(&msg.payload).ok()?;
-
-        if let Some(actor_id) = &cmd.actor_id {
-            if (inv.origin.is_actor() && inv.origin.public_key != actor_id.as_ref())
-                || (inv.target.is_actor() && inv.target.public_key != actor_id.as_ref())
-            {
-                return None;
-            }
+        // lattice.component.wrpc.0.0.1.operation.function
```

Comment:
So how do we handle these extra dots in the version exactly? A valid version could be something like `0.0.1-alpha.1` right?

Reply:
I don't think we considered non-numeric versions for the protocol, but I'm unsure. @rvolosatovs may have thoughts here, but afaik it was only planned to me Major.Minor.Patch